### PR TITLE
Add instructions to Easegress

### PR DESCRIPTION
**Pull request by LokiWager**
## BACKGROUND
* Add information about the background and reason for the change.
https://github.com/digitalocean/marketplace-kubernetes/pull/477
Add instructions to Easegress.

-----------------------------------------------------------------------

## Changes
* List the changes that you made

- add `stacks/easegress/README.md`
- add `stacks/easegress/imgs/architecture.png`
- add `stacks/easegress/values.yml`

-----------------------------------------------------------------------

## Checklist
- [x] review the [contributing doc](https://github.com/digitalocean/marketplace-kubernetes/blob/master/CONTRIBUTING.md) with steps for both adding or updating your application (if applicable)
------------------------------------------------------------------------

Reviewer: @marketplace-eng

### CODE REVIEW

File: stacks/easegress/README.md [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+* NOTE:
+
+You can get the `NODE_IP` and `NODE_PORT` by running the following command:
+
+```text
+export NODE_PORT=$(kubectl get --namespace easegress -o jsonpath="{.spec.ports[0].nodePort}" services easegress)
+export NODE_IP=$(kubectl get nodes --namespace easegress -o jsonpath="{.items[0].status.addresses[0].address}")
+echo http://$NODE_IP:$NODE_PORT
+
+http://10.108.162.15:30780
```
Comment (msannan2) [inline stacks/easegress/README.md:276]: Have this been tested DOKS? Is the service accessible with the NODE_IP or does it require a LoadBalancer as well?

**Review by msannan2 — CHANGES_REQUESTED** [review]

**Review by bikram20 — APPROVED** [review]
Only content changes. No engg review needed.

**Comment by LokiWager** [comment]
I’m sorry, I didn’t receive the github notification. I will make the necessary changes as soon as possible. @bikram20 thank you.

**Review by msannan2 — APPROVED** [review]
LGTM

**Pull request merged**